OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs

```diff
@@ -63,29 +77,21 @@ public void Run(ModData modData, string[] args)
 
 				foreach (var testMap in maps)
 				{
-					if (testMap != null)
-					{
-						Console.WriteLine("Testing map: {0}".F(testMap.Title));
-						testMap.PreloadRules();
-					}
-					else
-					{
-						Console.WriteLine("Testing mod: {0}".F(Game.ModData.Manifest.Mod.Title));
-					}
+					Console.WriteLine("Testing map: {0}".F(testMap.Title));
+					testMap.PreloadRules();
+					if (testMap.RuleDefinitions.Any() || testMap.VoiceDefinitions.Any() || testMap.WeaponDefinitions.Any())
+						CheckGameRules(testMap.Rules, testMap);
 
-					foreach (var customPassType in Game.ModData.ObjectCreator
-						.GetTypesImplementing<ILintPass>())
+					foreach (var customMapPassType in Game.ModData.ObjectCreator.GetTypesImplementing<ILintMapPass>())
```

Comment:
That isn't really describing anything. It already says `customMapPassType` and `ILintMapPass`. I also don't want to say when to use these, because people may come up with creative lint checks in the future.